sdk/metric/meter_test.go

```diff
@@ -2427,3 +2427,19 @@ func TestDuplicateInstrumentCreation(t *testing.T) {
 		})
 	}
 }
+
+func TestMeterProviderDelegation(t *testing.T) {
```

Comment:
Actually, can you move this to internal/global/meter_test.go?  And use the functions in global (global.MeterProvider, globlal.SetErrorHandler instead of the "otel" import.

Reply:
I tried moving it but my test is dependent on the NewMeterProvider() that is in the sdk/metric/provider.go to show the issue that we were seeing. Is there another meterprovider that would work from global?

Reply:
Are you able to use noop.NewMeterProvider() without pulling in any additional dependencies?

Reply:
`go.opentelemetry.io/otel/metric/noop`

Reply:
the noop.NewMetrerProvider doesn't have a delegate and skips over the original bug

Reply:
ok, lets keep it here.